node/coinstacks/litecoin/pulumi/index.ts

```diff
@@ -17,6 +24,7 @@ export = async (): Promise<Outputs> => {
   const asset = config.network !== 'mainnet' ? `${coinstack}-${config.network}` : coinstack
   const outputs: Outputs = {}
   const provider = new k8s.Provider('kube-provider', { kubeconfig })
+  const snapshots = await new VolumeSnapshotClient(kubeconfig, namespace).getVolumeSnapshots(asset)
```

Comment:
Add to all coinstacks? or still want to do some extra validation first?

Reply:
plan is validate against litecoin and then update remaining coinstacks.

litecoin:
- manual snapshot
- deploy and wait for reaper cron to snapshot
- scale up dev to test restore
- scale back down and update remaining coinstacks